src/init.cpp

```diff
@@ -404,6 +409,10 @@ void SetupServerArgs()
     hidden_args.emplace_back("-sysperms");
 #endif
     gArgs.AddArg("-txindex", strprintf("Maintain a full transaction index, used by the getrawtransaction rpc call (default: %u)", DEFAULT_TXINDEX), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-blockfilterindex=<type>",
+                 strprintf("Maintain an index of compact filters by block (default: %u, values: %s).", 0, ListBlockFilterTypes()) +
```

Comment:
Please define the default in a single location (ie, not both here and GetArg calls), and use %s in case it is a string.